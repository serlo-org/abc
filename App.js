export default (__DEV__
  ? require('./storybook').default
  : require('./src').defaut);
