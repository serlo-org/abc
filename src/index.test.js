import React from 'react';
import renderer from 'react-test-renderer';

import App from '.';

it('renders without crashing', () => {
  const tree = renderer.create(<App />);
  expect(tree).toMatchSnapshot();
});
