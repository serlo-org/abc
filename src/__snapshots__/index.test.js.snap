// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  disabled={false}
  ellipsizeMode="tail"
>
  Welcome to Serlo ABC
</Text>
`;
